dev/integration_tests/flutter_gallery/analysis_options.yaml

```diff
@@ -6,3 +6,5 @@ include: ../../../analysis_options.yaml
 analyzer:
   exclude:
     - build/**
+  enable-experiment:
```

Comment:
With the dart SDK constraint set to `>=2.12.0-0 < 3` this shouldn't be necessary.